### PR TITLE
2688 table flippa

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

- Add Table component to ui components
- Add Tables to lesmis

---

### Screenshots:
See issue

### CODE REVIEW

File: packages/lesmis/src/views/MapView.js [hunk 1]
```diff
@@ -86,14 +86,11 @@ export const MapView = () => {
     setSelectedOverlay,
   } = useMapOverlayReportData({ entityCode, year: selectedYear });
 
-  const handleLocationChange = useCallback(
-    (map, bounds) => {
-      const mapBoxBounds = Leaflet.latLngBounds(bounds);
-      const maxBounds = mapBoxBounds.pad(2);
-      map.setMaxBounds(maxBounds);
-    },
-    [Leaflet],
```
Comment (tcaiger) [inline packages/lesmis/src/views/MapView.js:95]: The console warning says to remove this dependency

File: packages/ui-components/src/components/Chart/Table.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  }
+
+  // table body
+  tbody {
+    th {
+      font-weight: 500;
+    }
+
+    tr {
+      &:nth-of-type(odd) {
```
Comment (edmofro) [inline packages/ui-components/src/components/Chart/Table.js:53]: This is a cool css selector I didn't know about!

File: packages/ui-components/src/components/Chart/Table.js [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+  left: calc(50%);
+`;
+
+const getColumns = rawData => {
+  const columns = [];
+  rawData.forEach(row => {
+    Object.keys(row)
+      .filter(key => key !== 'name' && key !== 'timestamp')
+      .forEach(key => {
+        // add the key as a table colunm but filter out object values such as metadata
```
Comment (edmofro) [inline packages/ui-components/src/components/Chart/Table.js:101]:
```suggestion
        // add the key as a table column but filter out object values such as metadata
```

File: packages/ui-components/src/components/Chart/Table.js [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+        }
+      });
+  });
+
+  return columns;
+};
+
+// For the rowData, ignore labelType and use percentage instead of fractionAndPercentage as
+// we don't want to show multiple values a table cell
+const parseValueType = valueType =>
```
Comment (edmofro) [inline packages/ui-components/src/components/Chart/Table.js:113]:
```suggestion
const sanitizeValueType = valueType =>
```

**Review by edmofro — APPROVED** [review]
Looks great! Nice, simple solution :-)

File: packages/ui-components/src/components/Chart/Table.js (hunk 2 above)
Comment (tcaiger) [reply packages/ui-components/src/components/Chart/Table.js:53]: the dark art of css 🧙

**Pull request merged**